docs/guides/machine_pool-2.md

```diff
@@ -0,0 +1,36 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "rhcs_machine_pool Resource - terraform-provider-rhcs"
+subcategory: ""
+description: |-
+  Machine pool.
+---
+
+# RHCS machine pool resource
+ The following endpoints are available for creating machine pools using the RHCS Terraform provider.
+
+## Machine pool parameters
+| Resource | Input Type |   Description   |
+|----------|------------|-----------------|
+| `cluster` | String | **Required**; This String variable identifies your cluster. You can use the cluster name or the cluster ID. |
+| `machine_type` | String | **Required**; This String variable identifies the machine type used by the nodes, for example `r5.xlarge`. Use the `rhcs_machine_types` data source to find the possible values. |
+| `name` | String | **Required**; This String variable is the name of the machine pool. Must consist of lower-case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character. |
+| `autoscaling_enabled`| Boolean | **Optional**; This Boolean variable enables or disables autoscaling. This variable requires you to set a maximum and minimum replicas range using the `max_replicas` and `min_replicas` variables. |
```

Comment:
I think better to mention here that it should also be assigned with min_replica and max_replica